beginner/apps/src/bin/blinky.rs

```diff
@@ -19,7 +20,7 @@ fn main() -> ! {
     for _ in 0..10 {
         led.toggle();
         timer.wait(Duration::from_secs(1));
-        log::info!("LED toggled at {:?}", dk::uptime());
+        defmt::debug!("LED toggled at {:?}", dk::uptime());
```

Comment:
Is it intended to change the log-level here? (mostly you are not)

Reply:
good catch!